script-based-workflows/R/plan.R

```diff
@@ -0,0 +1,18 @@
+
+#Load the scripts into functions via `code_to_function()`
+
+load_data <- code_to_function("R/01_data.R")
+do_munge <- code_to_function("R/02_munge.R")
+do_histogram <- code_to_function("R/03_histogram.R")
+do_regression <- code_to_function("R/04_regression.R")
+generate_report <- code_to_function("R/05_report.R")
```

Comment:
Could we put this in a `functions.R` file?

Reply:
Is that what we plan to suggest as an approach for script-based workflows? Adding a `functions.R` script to the process to separate out the building of the script-functions.

My argument for keeping them in the plan is that they are really not functions but representations of the scripts and having a single file that is really the `drake`-ification of the workflow seems more in line with the idea of in [Jenny's comment](https://github.com/ropenscilabs/drake-manual/issues/41#issuecomment-427721095). It also helps draw the connection between the scripts and the drake plan in my opinion.

Reply:
Yeah, the functions from `code_to_function()` are not going to be very well crafted (or really crafted at all for that matter). But we still do use them as functions in the plan, and we ultimately want to teach users how to create better functions for `drake`. And [`packages.R`/`functions.R`/`plan.R` is such a successful and ubiquitous trifecta](https://ropenscilabs.github.io/drake-manual/projects.html#code-files) that conformity is good. So I think a `functions.R` will spark understanding and set people on the right path.

Reply:
And I do not think it necessarily opposes https://github.com/ropenscilabs/drake-manual/issues/41#issuecomment-427721095.

Reply:
No problem. Since this will be the example for moving to script-based workflows, that makes sense to try to promote the trifecta.

Reply:
Thanks!